jslack-lightning/src/main/java/com/github/seratch/jslack/lightning/request/builtin/EventRequest.java

```diff
@@ -23,6 +23,7 @@ public EventRequest(
         this.headers = headers;
         JsonObject payload = GsonFactory.createSnakeCase().fromJson(requestBody, JsonElement.class).getAsJsonObject();
         this.eventType = payload.get("event").getAsJsonObject().get("type").getAsString();
+        this.getContext().setTeamId(payload.get("team_id").getAsString());
```

Comment:
We can set enterprise_id too. I'll add it after merging this PR.

Reply:
Good call.